src/giget.ts

```diff
@@ -71,7 +71,12 @@ export async function downloadTemplate (input: string, options: DownloadTemplate
   if (!options.offline) {
     await mkdir(dirname(tarPath), { recursive: true });
     const s = Date.now();
-    await download(template.tar, tarPath, { headers: template.headers }).catch((error) => {
+    await download(template.tar, tarPath, {
+      headers: {
+        ...template.headers,
+        Authorization: options.auth ? `Bearer ${options.auth}` : undefined,
```

Comment:
Template (resolved by registry) might override this header i think would be better to swap. Also using lower case `authorization` for consistency

Reply:
Hey @pi0 thanks for the quick response! i fixed it in latest push!